azb2c-with-parcel.md

```diff
@@ -0,0 +1,106 @@
+---
```

Comment:
Overall I have the impression you write with german grammar which makes me feel weird while reading the text. I suggest you cross-check the sentences with deeply or google translate. This can help avoid it.